examples/plotting_with_geoplot.py

```diff
@@ -14,75 +14,77 @@
 
 First we'll load in the data using GeoPandas.
 """
-import geopandas
-
-path = geopandas.datasets.get_path('naturalearth_lowres')
-df = geopandas.read_file(path)
-# Add a column we'll use later
-df['gdp_pp'] = df['gdp_md_est'] / df['pop_est']
-
-boroughs = geopandas.read_file(geopandas.datasets.get_path('nybb')).to_crs(epsg='4326')
-injurious_collisions = geopandas.read_file(
-    "https://github.com/ResidentMario/geoplot-data/raw/master/nyc-injurious-collisions.geojson")
+import geopandas as gpd
+import geoplot as gplt
 
+world = gpd.read_file(gpd.datasets.get_path('naturalearth_lowres'))
+boroughs = gpd.read_file(gplt.datasets.get_path('nyc_boroughs'))
+collisions = gpd.read_file(gplt.datasets.get_path('nyc_injurious_collisions'))
 
 ###############################################################################
 # Plotting with Geoplot
 # =====================
 #
 # We start out by replicating the basic GeoPandas world plot using Geoplot.
-import geoplot
-
-geoplot.polyplot(df, figsize=(8, 4))
+gplt.polyplot(world, figsize=(8, 4))
 
 ###############################################################################
 # Geoplot can re-project data into any of the map projections provided by
 # CartoPy (see the list
 # `here <http://scitools.org.uk/cartopy/docs/latest/crs/projections.html>`_).
 
 import geoplot.crs as gcrs
-ax = geoplot.polyplot(df, projection=gcrs.Orthographic(), figsize=(8, 4))
-ax.set_global()
+ax = gplt.polyplot(world, projection=gcrs.Orthographic(), figsize=(8, 4))
 ax.outline_patch.set_visible(True)
```

Comment:
Can you just make a tiny comment explaining what this line does?

Reply:
:+1: